src/widgets/about/about.data.tsx

```diff
@@ -6,11 +6,12 @@ import personIcon from '@/shared/assets/icons/person-icon.webp';
 import planetIcon from '@/shared/assets/icons/planet.webp';
 import type { CourseNames } from '@/shared/data/communication.data';
 import { DISCORD_LINKS } from '@/shared/data/communication.data';
+import { LinkCustom } from '@/shared/ui/link-custom';
 
 type AboutInfo = {
   id: number;
   title: string;
-  info: string;
+  info: string | JSX.Element;
```

Comment:
Please prefer to use ReactNode instead of JSX.Element whether it's possible

https://www.totaltypescript.com/jsx-element-vs-react-reactnode

```suggestion
  info: string | ReactNode;
```

<img width="608" alt="Screenshot 2024-08-30 at 22 47 42" src="https://github.com/user-attachments/assets/97ab8073-f928-4f40-a338-2b3650e5d826">

Reply:
thanks for article, changed.